### PR TITLE
CBI-81: maximize disk space for test github action

### PR DESCRIPTION
**Pull request by andrii-maglovanyi**

### CODE REVIEW

File: .github/workflows/run-tests-1.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        with:
+          root-reserve-mb: 512
+          swap-size-mb: 1024
+          remove-dotnet: 'true'
+      - uses: actions/checkout@v2
+      - name: 'Checkout submodules'
+        run: 'git submodule update --init --recursive'     
+      - name: 'Current disk space'
+        run: df -Ph . | tail -1 | awk '{print $4}'
+      - name: 'Clean up 1'
```
Comment (AndreiNavoichyk) [inline .github/workflows/run-tests-1.yml]: Do we need to clean up now?

File: .github/workflows/run-tests-1.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        run: sudo rm -rf "/usr/local/share/boost"
+      - name: 'Clean up 2'
```
Comment (AndreiNavoichyk) [inline .github/workflows/run-tests-1.yml]: Do we need clean up now?

File: .github/workflows/run-tests-2.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+name: 'Run tests 2'
```
Comment (AndreiNavoichyk) [inline .github/workflows/run-tests-2.yml]: Let's combine test-1 and test-2. It was split to test the theory that they can run successfully separately.

File: .github/workflows/run-tests-1.yml (hunk 1 above)
Comment (andrii-maglovanyi) [reply .github/workflows/run-tests-1.yml]: No, probably we don't

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**